Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -344,7 +387,9 @@ extension ContentBlockerHelper {
                     return
                 }
                 self.loadJsonFromBundle(forResource: filename) { jsonString in
-                    self.ruleStore.compileContentRuleList(forIdentifier: filename, encodedContentRuleList: jsonString) { _, _ in
+                    var str = jsonString
+                    str.insert(contentsOf: self.whitelistJSON(), at: str.index(str.endIndex, offsetBy: -1) )
```

Comment:
Whitelisted domains go at the end of a rule set.

Reply:
Should definitely be more efficient than de-serializing/serializing the JSON, but it feels a little fragile. Looks like `whitelistJSON()` always returns a leading `,` unless `ContentBlockerHelper.whitelistedDomains` is empty. I suppose that's ok as long as there is at least one rule to begin with.